sandbox/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+project(Sandbox)
+
+find_package(
+  Qt5
+  COMPONENTS Core Quick QuickControls2
+  REQUIRED)
+
+add_executable(
+  ${PROJECT_NAME}
+  main.cpp
+  handler.cpp
+  sandboxapp.cpp
+  spellchecker.cpp
+  qml.qrc
+  ../src/statusq.qrc
+  ../src/assets.qrc)
```

Comment:
Please use `qt_add_big_resources` for the qrc files; otherwise this will take ages to recompile

Reply:
Yup :)